docroot/themes/custom/uids_base/theme-settings.php

```diff
@@ -21,6 +24,24 @@ function uids_base_form_system_theme_settings_alter(&$form, FormStateInterface $
     '#open' => TRUE,
     '#tree' => TRUE,
   ];
+  $form['header']['type'] = [
+    '#type' => 'select',
+    '#title' => t('Site name display'),
+    '#description' => t('Select an option'),
+    '#options' => [
+      'inline' => t('Display inline with the IOWA bar'),
+      'below' => t('Display below the IOWA bar'),
+    ],
+    '#default_value' => theme_get_setting('header.type'),
+  ];
+  if ($has_parent) {
+    $test = Url::fromUri('internal:/admin/config/system/site-information');
+    $form['header']['type']['#disabled'] = TRUE;
+    $form['header']['type']['#default_value'] = 'below';
+    $form['header']['type']['#description'] = t('This option is disabled because a parent organization was set on the <a href=":site-settings-page">site settings page</a>. When you have a parent organization, your site name will <em>always</em> display on the line below. You will need to remove the parent organization information to select another option.', [
+      ':site-settings-page' => Url::fromUri('internal:/admin/config/system/site-information')->toString(),
```

Comment:
Better to use route than string path `Url::fromRoute('system.site_information_settings'...`